### ISSUE TITLE
Define exit codes for post-job failures and report them to dashboard and crab status

### ISSUE THREAD

**Issue by AndresTanasijczuk** [dmwm/CRABServer#4831]

**Comment by belforte** [comment]
@AndresTanasijczuk 
since the related PR was merged.. can this be closed ?

**Comment by AndresTanasijczuk** [comment]
No, this issue has not been solved. The above pull request #4820 is not for this issue.
What this issue refers to is the following: 
* Currently any failure that happens in the postjob cases either a "fatal" or a "recoverable" error.    
* The Post-job exit code is 2 and 1 respectively.
* The user sees the job status as "post-processing failed" both in dashboard and 'crab status'.
* We would like to have a range of post-job exit codes to indicate what the failure was, and show this exit code in dashboard and 'crab status'.

**Comment by belforte** [comment]
ok. Does not look urgent though.
